extensions/job_management/standard/requirements/job-management/definition/REQ_get-op.adoc

```diff
@@ -0,0 +1,9 @@
+[[req_job-management_definition_get-op]]
+[requirement]
+====
+[%metadata]
+label:: /req/job-management/definition-get-op
+part:: For every jobs (using method: POST on path: /jobs/{jobId}), the server SHALL support the HTTP GET operation at the path `/jobs/{jobId}/definition`.
```

Comment:
What is the definition meant to return? The process? (e.g. CWL or openEO UDP?)

We include that directly in GET /jobs/:id - I guess that's fine and we can add an additional endpoint, but maybe this is more an optional endpoint for cases where a definition can't be enbedded? And it can be explored via a link in GET /jobs/:id?

What is POST /jobs/:id?

Reply:
CWL or openEO UDP -> Yes

POST /jobs/:id -> typo

I'm fine with making this endpoint optional since servers can indeed return the definition directly and avoid the extra request. However, if that is the case, I suggest that a requirement says as such explicitly, or that providing a link reference is required (either pointing to the job itself or `/jobs/{jobId}/definition`). The important aspect is that the job definition can be found "somewhere".